Operations_on_polyhedra/include/CGAL/internal/Polyhedron_plane_clipping_3.h

```diff
@@ -131,7 +131,7 @@ Polyhedron clip_to_bbox(const Bbox_3& bbox, const Plane_3& plane)
     if (orientations[i2]==ON_ORIENTED_BOUNDARY) continue;
     if (orientations[i1]!=orientations[i2])
       intersection_points.push_back(
-        get<Point_3>(
+        boost::get<Point_3>(
```

Comment:
Go and read the SO question and answer. The compilation error has nothing to with ADL, but with the parsing of the expression `get<Point_3>`: even before the name lookup rules are used (including ADL), the expression must be parse as a correct C++ expression. If the compiler has no "view", as regards the current namespace/scope, of a correct `get` template function, then the parsing trigger a compilation error.

You can even use `using CGAL::get` (if it is defined in the included headers) even if in the end the argument-dependent lookup will conclude that `boost::get` is the right match.